README.md

```diff
@@ -1,234 +1,114 @@
-    alb.ingress.kubernetes.io/subnets: <VPC_PUBLIC_SUBNET>
-    alb.ingress.kubernetes.io/tags: env=dev
-    alb.ingress.kubernetes.io/target-type: ip
-  host: sombra-transcend.my-domain.com
-
-customer_service:
-  type: NodePort
-  port: 5039
-  # Annotations to add to the service account
-  annotations: {}
-
-customer_ingress:
-  enabled: true
-  className: alb
-  annotations:
-    alb.ingress.kubernetes.io/certificate-arn: <CERT_ARN>
-    alb.ingress.kubernetes.io/healthcheck-path: /health
-    alb.ingress.kubernetes.io/healthcheck-port: "5039"
-    alb.ingress.kubernetes.io/healthcheck-protocol: HTTP
-    alb.ingress.kubernetes.io/listen-ports: '[{"HTTPS": 5039}]'
-    alb.ingress.kubernetes.io/scheme: internal
-    alb.ingress.kubernetes.io/subnets: <VPC_PRIVATE_SUBNET>
-    alb.ingress.kubernetes.io/tags: env=dev
-    alb.ingress.kubernetes.io/target-type: ip
-  host: sombra-customer.my-domain.com
-
-envs:
-  - name: ORGANIZATION_URI
-    value: '<ORGANIZATION_URI>'
-  - name: EMPLOYEE_AUTHENTICATION_METHODS
-    value: 'transcend,session'
-  - name: DATA_SUBJECT_AUTHENTICATION_METHODS
-    value: 'transcend,session'
-
-envs_as_secret:
-  - name: INTERNAL_KEY_HASH 
-    value: '<INTERNAL_KEY_HASH>'
-  - name: JWT_ECDSA_KEY
-    value: '<JWT_ECDSA_KEY>'
-  - name: INTERNAL_KEY
-    value: '<INTERNAL_KEY>'
-
-```
-4.	Install the Sombra package.
+3. To configure the chart and provide environment variables, create a `values.yaml` file:
+
+    ```yaml
+    imageCredentials:
+      registry: docker.transcend.io
+      username: Transcend
+      password: '<TRANSCEND_API_TOKEN>'
+
+    transcend_service:
+      type: NodePort
+      port: 5042
+      # Annotations to add to the service account
+      annotations: {}
+
+    transcend_ingress:
+      enabled: true
+      className: alb
+      annotations:
+        alb.ingress.kubernetes.io/certificate-arn: <CERT_ARN>
+        alb.ingress.kubernetes.io/healthcheck-path: /health
+        alb.ingress.kubernetes.io/healthcheck-port: '5042'
+        alb.ingress.kubernetes.io/healthcheck-protocol: HTTP
+        alb.ingress.kubernetes.io/listen-ports: '[{"HTTPS": 5042}]'
+        alb.ingress.kubernetes.io/scheme: internet-facing
+        alb.ingress.kubernetes.io/subnets: <VPC_PUBLIC_SUBNET>
+        alb.ingress.kubernetes.io/tags: env=dev
+        alb.ingress.kubernetes.io/target-type: ip
+      host: sombra-transcend.my-domain.com
+
+    customer_service:
+      type: NodePort
+      port: 5039
+      # Annotations to add to the service account
+      annotations: {}
+
+    customer_ingress:
+      enabled: true
+      className: alb
+      annotations:
+        alb.ingress.kubernetes.io/certificate-arn: <CERT_ARN>
+        alb.ingress.kubernetes.io/healthcheck-path: /health
+        alb.ingress.kubernetes.io/healthcheck-port: '5039'
+        alb.ingress.kubernetes.io/healthcheck-protocol: HTTP
+        alb.ingress.kubernetes.io/listen-ports: '[{"HTTPS": 5039}]'
```

Comment:
Shouldn't the listener ports be HTTP for HTTP configurations?

Reply:
`listen-ports` here defining the ports on which Load balancer will listen which we mostly use HTTPS  to keep traffic b/w client and load balancer over TLS

Reply:
Nice, thanks for the explanation!